### ISSUE TITLE
DOTCOM-25391 - Accessibility: "Load more articles" buttons does not receive keyboard focus

### ISSUE THREAD

**Issue by avanishgandhi** [adobe/theblog#295]
**JIRA LINK** - https://jira.corp.adobe.com/browse/DOTCOM-25391?src=confmacro

**Context** 
Chrome Version 83.0.4103.116 on macOS. MacBook Pro running macOS Catalina 10.15.5

**URL**: https://blog.adobe.com/

**Reproduction Steps:**

Tab through the above page using keyboard.
Once you finish navigating through the cards, notice that 'load more articles' button is not receiving keyboard focus.
Expected behavior

This button, which is marked up as anchor tag, needs

role="button" so that screenreader users know that this is a button.
tabindex="0" for keyboard users. The anchor tag isn't interactive unless it has a href attribute.
So, we need to add button role and tabindex to define the semantics and keyboard behavior.
Now that it is defined as a button, make sure it can be activated using both Enter and Space key.
And make sure that this button has a focus outline when it receives keyboard focus.

**WCAG Success Criteria**
Keyboard

**Comment by davidnuescheler** [comment]
should be fixed on a staging...

https://staging--theblog--adobe.hlx.page/

**Comment by avanishgandhi** [comment]
Works in stage. When will this be push to prod? 

![image](https://user-images.githubusercontent.com/62259400/88416491-2ef74680-cd95-11ea-8151-ca69d79fa3a7.png)

**Comment by rofe** [comment]
This was merged to master yesterday. Closing.
